WINDOWS_GUI/UDS_Spec/uds_comm_spec.h

```diff
@@ -54,6 +54,11 @@ extern "C" {
 #define FBL_TRANSFER_DATA                                           (0x36)
 #define FBL_REQUEST_TRANSFER_EXIT                                   (0x37)
 
+/**
+ * Own
+ */
```

Comment:
I think something like value for our test asw would be a more descriptive comment than own